### PR TITLE
Prevent fatal error if doctrine is require but not available

### PR DESCRIPTION
**Pull request by lchrusciel**
Warning extraction from #43. 

BC break inside.

### CODE REVIEW

File: src/ApiTestCase.php [hunk 1]
```diff
@@ -292,7 +293,31 @@ protected function loadFixturesFromFile($source)
...
-        return $this->fixtureLoader->loadFiles($source);
+        return $this->getFixtureLoader()->loadFiles($source);
+    }
+
+    /**
+     * @return Fixtures
+     */
+    protected function getFixtureLoader()
+    {
+        if (is_null($this->fixtureLoader)) {
```
Comment (pamil) [inline src/ApiTestCase.php]: `is_null(X)` -> `null === X`?

File: src/ApiTestCase.php [hunk 2]
```diff
@@ -292,7 +293,31 @@ protected function loadFixturesFromFile($source)
...
+
+        return $this->fixtureLoader;
+    }
+
+    /**
+     * @return EntityManager
+     */
+    protected function getEntityManager()
+    {
+        if (is_null($this->entityManager) || !$this->entityManager->getConnection()->isConnected()) {
```
Comment (pamil) [inline src/ApiTestCase.php]: Same here.

**Comment by pamil** [comment]
Rebase needed :)

**Comment by lchrusciel** [comment]
@pamil done.

**Comment by pamil** [comment]
:+1:

**Pull request merged**